test/e2e/memtierd.test-suite/policy-age/n4c16/test01-softdirty/code.var.sh

```diff
@@ -0,0 +1,81 @@
+memtierd-setup
+
+# Test meme process which has 1G of memory and writes actively 300M
+# 700M+ should be moved to idlenumas and 300M should be moved to activenumas
+MEME_CGROUP=e2e-meme
+MEME_BS=1G MEME_BWC=1 MEME_BWS=300M MEME_MEMS=0 memtierd-meme-start
+sleep 2
+
+MEMTIERD_YAML="
+policy:
+  name: age
+  config: |
+    intervalms: 5000
+    pidwatcher:
+      name: cgroups
+      config: |
+        cgroups:
+          - /sys/fs/cgroup/${MEME_CGROUP}
+    idledurationms: 8000
+    idlenumas: [3]
+    activedurationms: 6000
+    activenumas: [1]
+    tracker:
+      name: softdirty
+      config: |
+        pagesinregion: 256
+        maxcountperregion: 0
+        scanintervalms: 4000
+        regionsupdatems: 0
+        skippageprob: 0
+        pagemapreadahead: 0
+    mover:
+      intervalms: 20
+      bandwidth: 200
+"
+memtierd-start
+
+next-round() {
+    local round_counter_var=$1
+    local round_counter_val=${!1}
+    local round_counter_max=$2
+    local round_delay=$3
+    if [[ "$round_counter_val" -ge "$round_counter_max" ]]; then
+        return 1
+    fi
+    eval "$round_counter_var=$(($round_counter_val + 1))"
+    sleep $round_delay
+    return 0
+}
+
+match-pageout() {
```

Comment:
The match function name, its variable names and message contents (echo, error) should not talk about pageout, because now we are moving pages from one NUMA node to another.